src/references/dataProtector/dataProtectorSharing/consume/consumeProtectedData.md

```diff
@@ -81,8 +82,9 @@ const consumeProtectedDataResult =
 
 **Type:** `AddressOrENS`
 
-Address or ENS of the iExec TEE iApp that will be used to consume the protected
-data. This iExec TEE iApp is the one that runs within an iExec worker.
+Address or Ethereum Name Service (ENS) of the iExec TEE iApp that will be used
+to consume the protected data. This iExec TEE iApp is the one that runs within
```

Comment:
The iExec TEE iApp , is duplicate, should say this iApp or the iExec App running in TEE